<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>SCHNAPPS_PLUGIN_SURFACE_MODELISATION_API</exportmacro>
 <class>Subdivision_Dialog</class>
 <widget class="QDialog" name="Subdivision_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subdivision</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="list_maps"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Position attribute :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_positionAttribute">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_loop">
       <property name="text">
        <string>Loop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_catmull_clark">
       <property name="text">
        <string>Catmull-Clark</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
